src/dtos/posts.dto.ts

```diff
@@ -0,0 +1,15 @@
+export type BasePostDto = {
+  readonly id: number;
+  readonly media?: string;
+  readonly userId: number;
+  readonly content: string;
+  readonly categoryName: string;
+};
+
+export type GetPostResponseDto = BasePostDto;
```

Comment:
Let's not create these DTO types now. I think it makes sense to create it when we are going to work with it